proto/satellite.proto

```diff
@@ -264,3 +264,47 @@ message SatOpRow {
     // check nulls_bitmask to differentiate between the two
     repeated bytes values = 2;
 }
+
+// A migration message, originating in Postgres, captured via event triggers,
+// propated to electric, converted from postgres to the equivalent sqlite
+// statement and inserted into the replication stream
+//
+// Each migration message includes the sql strings to execute on the satellite
+// client as well as metadata information about the resulting structure of the
+// changed tables.
+message SatMigration {
+    enum Type {
+        CREATE_TABLE = 0;
+        CREATE_INDEX = 1;
+        ALTER_ADD_COLUMN = 6;
+    }
+    message Stmt {
+        Type type = 1;
+        string sql = 2;
+    }
+    message Column {
+        string name = 1;
+        string sqlite_type = 2;
+        string pg_type = 3;
+    }
+    message ForeignKey {
```

Comment:
I wonder if we also need to add PrimaryKey information here?

Reply:
no idea. it's information we have at the point of constructing this message so easy enough to include. the maximal approach would say add it in case it's useful later